### DISCUSSION TITLE
Does Highres fix work (or is it needed) for Flux

### DISCUSSION THREAD

**Discussion by toyssamurai** [lllyasviel/stable-diffusion-webui-forge#1457] [Q&A]
Clearly, highres fix is doing something because I did get a higher res image, but if I set the denoising above, say 0.2, the image became overexposed. But anything lower, I would get an image that is like a double outline. 

I don't need it "fixed", I just need to know is the feature supposed to work?

**Comment by HMRMike** [comment]
Definitely works, and with some LORAs it's definitely needed or else they look terrible. 
In my case it does very little under 0.5 denoise. But I haven't seen any degradation overall.
For example 512x512 with 2x upscale (NMKD SIAX), 0.5 denoise.
![00103-123](https://github.com/user-attachments/assets/93981c92-41f4-449b-8f3b-49a6cf2d65f1) 
![00104-123](https://github.com/user-attachments/assets/23b28889-0f29-4cb5-9d70-abc8302cb408)

**Comment by stripealipe** [comment]
What upscaling model are you using? You should be getting nice results unless you go high on the denoise, which it seems you're not.

**Reply by toyssamurai** [reply]
I think I found out why -- I always use latent upscaling and that appears to be the problem. Too bad, though. I prefer to use latent upscaling as it tends to introdue artifacts that other people don't like, but inspire me to do something else to the artwork.
